### PR TITLE
Configure CSS nesting correctly

### PR DESCRIPTION
**Pull request by dem4ron**

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   plugins: [
     require('postcss-import'),
-    require('tailwindcss/nesting'),
+    require('tailwindcss/nesting')(require('postcss-nesting')),
```
Comment (iHiD) [inline postcss.config.js:4]: What does this line do?

Comment (dem4ron) [reply postcss.config.js:4]: following the [docs](https://tailwindcss.com/docs/using-with-preprocessors#nesting) you'd need to write it in an object format like this: 
```javascript
// postcss.config.js
module.exports = {
  plugins: {
    'postcss-import': {},
    'tailwindcss/nesting': 'postcss-nesting',
    tailwindcss: {},
    autoprefixer: {},
  }
}
```

but as it was already configured in this commonjs format, this is just the analogue of the above. It invokes the required `tailwindcss/nesting` plugin with the required `postcss-nesting` plugin as an argument.

**Review by ErikSchierboom — APPROVED** [review]
Does this remove the warning in the logs?

**Comment by dem4ron** [comment]
> Does this remove the warning in the logs?

yes

**Pull request merged**